repo_manager/gh/settings.py

```diff
@@ -66,14 +68,21 @@ def get_repo_value(setting_name: str, repo: Repository) -> Any | None:
     for setting_name in settings.dict().keys():
         repo_value = get_repo_value(setting_name, repo)
         settings_value = getattr(settings, setting_name)
-        # These don't seem to update if changed; may need to explore a different API call
-        if (setting_name == "enable_automated_security_fixes") | (setting_name == "enable_vulnerability_alerts"):
-            continue
+        if setting_name in [
+            "allow_squash_merge",
+            "allow_merge_commit",
+            "allow_rebase_merge",
+            "delete_branch_on_merge",
+            "enable_automated_security_fixes",
+            "enable_vulnerability_alerts",
+        ]:
+            if repo._requester.oauth_scopes is None:
```

Comment:
@andrewthetechie - This works with the old PAT types, like what you are using now, but the new PAT types always have oauth_scopes of NONE it seems...  This is where if i use a new PAT type or the github.token diffs get falsely displayed because the oauth denies access to these settings and incorrectly flags them as different because the oauth denial returns NONE instead of an error.